src/scripts/modules/list.js

```diff
@@ -69,16 +95,13 @@ function renderDestination(destination) {
   );
   // add captions to destination
   elDestination.appendChild(figcaption);
-
-  // events.subscribe("destinationAdded", addDestination);
-
   // Return the destination HTML
   return elDestination;
 }
 
 function render() {
   // Create DOM elements for each destination
-  destinationsList.forEach((destination) => {
+  [...destinationsList, ...storedList].forEach((destination) => {
     const elDestination = renderDestination(destination);
     gallery.appendChild(elDestination);
   });
```

Comment:
Following my last comment about `destinationUpdate` received the `newList`, there's a little big detail missing. Pick this up only after you addressed all the other comments.

With the current code, each time render is called, we are appending over and over again the list to the gallery, which means you'll see the items being duplicated. So we need to restore (clear) the gallery before appending the items.

<details>
<summary>Solution</summary>

```js
function render() {
  // Clean gallery with old DOM els to avoid duplication
  gallery.textContent = '';


  // Create DOM elements for each destination
  storedList.forEach((destination) => {
    const elDestination = renderDestination(destination);
    gallery.appendChild(elDestination);
  });
}
```

</details>